### PR TITLE
Upgrade to broccoli-persistent-filter

### PR DESCRIPTION
**Pull request by thoov**
@shinnn @stefanpenner

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "index.js"
   ],
   "scripts": {
-    "pretest": "eslint --config node_modules/@shinnn/eslintrc-node/rc.json --rule 'no-underscore-dangle: 0' index.js test/test.js",
```
Comment (shinnn) [inline package.json]: Why did you remove `pretest`?

File: index.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 'use strict';
 
 var path = require('path');
-
```
Comment (shinnn) [inline index.js]: Please do not remove this line.

File: package.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
...
+
```
Comment (shinnn) [inline package.json]: package.json shouldn't include blank lines.

File: package.json (hunk 1 above)
Comment (stefanpenner) [inline package.json]: @thoov ^

**Comment by shinnn** [comment]
Landed in https://github.com/shinnn/broccoli-clean-css/commit/b13e42d65c373aaca96b5db58f7a5100aa6fe4ff.

Thanks, @thoov!

**Comment by shinnn** [comment]
@thoov @stefanpenner 

I'm planning to release the next version as v2.0.0 which uses ES2015 syntax as possible and drop Node 0.x. Can I just go as planned, or should I still support Node v0.x?

**Comment by stefanpenner** [comment]
> or should I still support Node v0.x?

Ember-cli continues to support node 0.10.x (my employers default infrastructure is still tied to 0.10.x, so I will need to continue its support) I would prefer this library also remain 0.10.x+, otherwise we wont be able to upgrade.

**Comment by shinnn** [comment]
> otherwise we wont be able to upgrade.

Hmm, it seems that Ember-cli currently doesn't depend on broccoli-clean-css. https://github.com/ember-cli/ember-cli/commit/489f68b6a3bdca1bb4aadefb71c5ba75a6b22bbc#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L78

**Comment by stefanpenner** [comment]
it does via its dependency on the preprocessor registry https://github.com/ember-cli/ember-cli-preprocessor-registry/blob/master/package.json#L20 which includes csso as the default fallback: https://github.com/ember-cli/ember-cli-preprocessor-registry/blob/2fc5043a871cd7491f3cd374c10b0f08c0de7509/preprocessors.js#L116-L117

**Comment by shinnn** [comment]
OK, I released the latest version as v1.1.0 which still supports Node 0.10.x.

**Comment by stefanpenner** [comment]
@shinnn thank you kindly.
